### PR TITLE
docs: updated the dev slack channel name issue #30

### PR DESCRIPTION
**Pull request by mrajen27**
docs: updated the dev slack channel name.

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -8,7 +8,7 @@ to help deppy contributors.
 ## Communication
 
 - Email: [operator-framework-olm-dev](mailto:operator-framework-olm-dev@googlegroups.com)
-- Slack: [#rukpak-dev](https://kubernetes.slack.com/archives/C038B7MF75M)
+- Slack: [#deppy-dev](https://kubernetes.slack.com/archives/C038B7MF75M)
```
Comment (akihikokuroda) [inline CONTRIBUTING.MD:11]: The channel id seems C045L6WK27N instead of C038B7MF75M.

Comment (timflannagan) [reply CONTRIBUTING.MD:11]: Yup, https://kubernetes.slack.com/archives/C045L6WK27N appears to be the new link.

Comment (mrajen27) [reply CONTRIBUTING.MD]: thankyou for providing the correct channel-id , i have updated my PR.

**Review by akihikokuroda — APPROVED** [review]
LGTM.  Thanks!

**Review by timflannagan — APPROVED** [review]

**Pull request merged**